tests/testthat/test-diagnose-design.R

```diff
@@ -198,3 +198,15 @@ test_that("default diagnosands work", {
   diag <- diagnose_design(sims, sims = 5, bootstrap_sims = FALSE)
 })
 
+
+test_that("with and without term",{
+  library(DeclareDesign)
+  library(DesignLibrary)
```

Comment:
Generally we don't want library inside a test, use the skip_if_not_installed to check, and then do DesignLibrary::simple_factorial_designer.

Reply:
great, thank you

Reply:
done